### DISCUSSION TITLE
Horizontal and Vertical Door

### DISCUSSION THREAD

**Discussion by heymatt18** [OndrejNepozitek/Edgar-Unity#110] [General]
Hi there, 

I have some assets that I'm trying to use that have walls that are 3 units in height. Currently from what I can tell, I can't have a horizontal door that is any more than 1 unit in height, but I would like to be able to have doors spawning on the top and bottom of my room templates. Is there a way to get around this/a location I can begin working in to modify the scripts myself? I'm pretty committed to the art style of my assets (and I may or may not have put money towards them), so I would really like to be able to use them.

Thanks!

**Comment by OndrejNepozitek** [comment]
Hello, yes, it's possible. Please take a look at [this section](https://ondrejnepozitek.github.io/Edgar-Unity/docs/other/faq/#how-to-deal-with-wider-walls) of the FAQ page. 

Let me know if that helps!

**Reply by heymatt18** [reply]
Totally missed that in the documentation. Thanks!
